Scripts/Game/ODIN/Editor/Containers/Attributes/ODIN_ToggleDamageVehicleAttribute.c

```diff
@@ -0,0 +1,46 @@
+/**
+Entity Attribute for toggle damage on unit
+*/
+[BaseContainerProps(), SCR_BaseEditorAttributeCustomTitle()]
+class ODIN_ToggleDamageVehicleAttribute : SCR_BaseEditorAttribute
+{
+	override SCR_BaseEditorAttributeVar ReadVariable(Managed item, SCR_AttributesManagerEditorComponent manager)
+	{
+		
+		SCR_EditableEntityComponent editableEntity = SCR_EditableEntityComponent.Cast(item);
+		if (!editableEntity) 
+			return null;
+		
+		IEntity owner =  editableEntity.GetOwner();
+		if (!owner) 
+			return null;
+		
+		if (SCR_ChimeraCharacter.Cast(owner))
```

Comment:
Check that it cannot be casted to `Vehicle` instead. Might be more sensible if we get editable destructible static objects in future.

Reply:
Would have to add check that it isn't a "group", "waypoint" or "logic" then too? As they are not vehicles, but also not something where damage can be toggled?

Reply:
There's no need for additional checks

Reply:
My bad. I somehow misunderstood what you meant. Good catch.